src/main/java/com/wishlist/model/Category.java

```diff
@@ -0,0 +1,25 @@
+package com.wishlist.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import com.wishlist.enums.Categories;
+import javax.persistence.*;
+import java.util.Collection;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name="categories")
+
+public class Category {
```

Comment:
What does this class represent? After reading the implementation it is quite hard to guess...